Source/MissionSystem/Private/MSMission.cpp

```diff
@@ -120,11 +122,23 @@ bool UMSMission::IsComplete() const
 }
 
 #if !( UE_BUILD_SHIPPING || UE_BUILD_TEST )
-void UMSMission::DumpObjectives( FOutputDevice & output_device )
+void UMSMission::DumpMission( FOutputDevice & output_device )
 {
+    const FString status = IsComplete() ? "Completed" : ( bIsCancelled ? "Cancelled" : "OnGoing" );
+
+    output_device.Logf( ELogVerbosity::Verbose,
+        TEXT( " * Mission : %s - Status: %s" ),
+        *GetNameSafe( GetMissionData() ),
+        *status );
+
     for ( const auto * objective : Objectives )
     {
-        output_device.Logf( ELogVerbosity::Verbose, TEXT( "   - Objective : %s - Complete : %s" ), *GetNameSafe( objective ), *UKismetStringLibrary::Conv_BoolToString( objective->IsComplete() ) );
+        const FString objective_status = objective->IsComplete() ? "Completed" : ( objective->IsCancelled() ? "Cancelled" : "OnGoing" );

```

Comment:
you can create a lambda to avoid duplication of the code

Reply:
Done.